js/components/cohortbuilder/CriteriaTypes/Criteria.js

```diff
@@ -1,10 +1,9 @@
-define(['require', 'knockout', '../InputTypes/Range','conceptpicker/InputTypes/Concept', '../InputTypes/Text', '../CriteriaGroup'], function (require, ko, Range, Concept, Text, CriteriaGroup) {
+define(['knockout', '../InputTypes/Range','conceptpicker/InputTypes/Concept', '../InputTypes/Text', '../CriteriaGroup'], function (ko, Range, Concept, Text, CriteriaGroup) {
 
 	function Criteria (data, conceptSets) {
 		var self = this;
 		var data = data || {};
 		
-		var CriteriaGroup = require("../CriteriaGroup");
```

Comment:
This is the circular reference i was referring to. Without doing it this way, the CriteriaGroup will be null as the require parameter, and you need to do a require() again later. I will pull the branch and verify, however.